extras/parse_travis_log.py

```diff
@@ -360,6 +360,19 @@ def makebuild_summary(log_filename, msg_make_section_start,
     number_of_error_msgs = 0
     number_of_warning_msgs = 0
     in_make_section = False
+    if environ['xNEST_BUILD_TYPE'] == 'MINIMAL':
+        expected_warnings = 8
+    elif environ['xNEST_BUILD_TYPE'] == 'MPI_ONLY':
+        expected_warnings = 263
+    elif environ['xNEST_BUILD_TYPE'] == 'OPENMP_ONLY':
+        expected_warnings = 8
+    elif environ['xNEST_BUILD_TYPE'] == 'FULL':
+        expected_warnings = 4006
+    elif environ['xNEST_BUILD_TYPE'] == 'FULL_NO_EXTERNAL_FEATURES':
+        expected_warnings = 8
+    else:
+        expected_warnings = 0  # Set to 0 if none of the above build-types, to not crash the script
+
```

Comment:
From a code design point of view, wouldn't it be better to pass the content of `NEST_BUILD_TYPE` as a parameter to `parse_travis_log.py`? This would expose the logical dependency between the `.travis.yml` file and the log parser and reduce the risk of someone changing the name and accidentally breaking the log parser.

Reply:
Good idea, I've changed it.